src/few/summation/directmodesum.py

```diff
@@ -121,6 +124,7 @@ def sum(
         # waveform sum where m < 0
         w2 = self.xp.sum(
             (m_arr[self.xp.newaxis, inds] > 0)
+            * (-1) ** l_arr[self.xp.newaxis, inds]
```

Comment:
Why do we introduce this here? Isn't the $(-1)^l$ operation taken into account in the mode amplitudes now?

Reply:
Needed due to assumptions of the CUDA kernel.